invenio_app_rdm/static_pages/static_pages.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Static pages."""
+import os
+from pathlib import Path
+
+from flask_webpackext import current_webpack
+from invenio_db import db
+from invenio_pages import Page
+
+
+class StaticPages:
+    """Static pages."""
+
+    def page_data(self, page):
+        """Returns content of template."""
+        assets_path = current_webpack.project.path
+        overriden_page = Path(
+            assets_path + os.path.join("/templates/static-pages", page)
+        )
+        if overriden_page.exists():
+            return (
+                Path(assets_path + os.path.join("/templates/static-pages", page))
+                .read_bytes()
+                .decode("utf8")
+            )
+
+        return (
+            Path(Path(__file__).parent.as_posix() + os.path.join("/templates", page))
+            .read_bytes()
+            .decode("utf8")
+        )
+
+    def run(self, force=False):
+        """Run the creation of Static Pages."""
+        pages = [
+            Page(
```

Comment:
minor: if we use the `FixtureMixin` described above, these entries could go in a `pages.yml` file, and the HTML file could be loaded from a `pages/` folder or similar.